src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.nuspec

```diff
@@ -2,21 +2,19 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>FakeItEasy.Analyzer</id>
-    <version>$version$</version>
+    <version>5.0.0</version>
     <title>FakeItEasy.Analyzer</title>
     <authors>Thomas Levesque, FakeItEasy contributors</authors>
-    <description>A convenience package that installs the FakeItEasy Analyzer for both C# and Visual Basic. Clients using FakeItEasy from only one of the two languages should prefer FakeItEasy.Analyzer.CSharp or FakeItEasy.Analyzer.VisualBasic. Provides diagnostic analyzers to warn about incorrect usage of FakeItEasy. Works in Visual Studio 2015 Update 1 or later.</description>
+    <description>DO NO INSTALL THIS PACKAGE. It's no longer maintained and does nothing. Use one of the language-specific analyzer packages instead: FakeItEasy.Analyzer.CSharp or FakeItEasy.Analyzer.VisualBasic.</description>

```

Comment:
I wish I'd noticed that "DO NO" should be "DO NOT" the first time around!

Reply:
oops

Reply:
Fixed